plugins/buildingplan/itemfilter.cpp

```diff
@@ -35,7 +35,7 @@ bool ItemFilter::isEmpty() const {
             && materials.empty();
 }
 
-static bool deserializeMaterialMask(string ser, df::dfhack_material_category mat_mask) {
+static bool deserializeMaterialMask(const string& ser, df::dfhack_material_category& mat_mask) {
```

Comment:
all other changes in this PR are incidental. This is the functionally different line.